qiskit/quantum_info/states/densitymatrix.py

```diff
@@ -133,8 +139,8 @@ def draw(self, output=None, max_size=(16, 16), dims=None, prefix='', **drawer_ar
 
         Args:
             output (str): Select the output method to use for drawing the
-                circuit. Valid choices are ``text``, ``latex``, ``latex_source``,
-                ``qsphere``, ``hinton``, or ``bloch``. Default is ``auto``.
+                circuit. Valid choices are `repr`, `text`, `latex`, `latex_source`,
```

Comment:
Would be good for the documenation of this function should state that the default comes from qiskit conf and how to change it.

Reply:
Thanks, done!